scripts/integration/mqtt/test.yaml

```
@@ -0,0 +1,11 @@
+features:
+- mqtt-integration-tests
+
+test_filter: '::mqtt::'
+
+matrix:
+  version: ['5.0.15']
+
+paths:
+- "src/internal_events/mqtt.rs"
+- "src/sinks/mqtt/**"
```

In general since all sinks use the `prelude.rs` which heavily leverages `util` module, we want to run all sink integration tests when that changes.

```suggestion
- "src/sinks/mqtt/**"
- "src/sinks/util/**"
```